.changeset/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+    "changelog": [
+        "@changesets/changelog-github",
+        {
+            "repo": "adobe/spectrum-web-components"
+        }
+    ],
+    "commit": false,
+    "fixed": [],
+    "linked": [],
```

Comment:
Do you need to look into the concept of ["linked" packages](https://github.com/changesets/changesets/blob/main/docs/linked-packages.md) for this monorepo?

I didn't look too deeply at it for Spectrum CSS because all of our packages are published and versioned individually, but your case may be different.

Reply:
I think for SWC use case adding [fixed packages](https://github.com/changesets/changesets/blob/main/docs/fixed-packages.md) makes sense since we would want to bump the versions of all the packages on a release.